changelog.d/20240629_120748_codewithemad_cleanup_flag.md

```diff
@@ -0,0 +1 @@
+- 💥[Feature] Enhance your workflow with the new `-c` or `--clean` option for the `tutor config save` command! This feature allows you to clean your environment before each save, ensuring that all files and directories within the `env/` folder are deleted, providing you with a fresh environment each time. (by @CodeWithEmad)
```

Comment:
It's not a breaking change. The feature is only accessible behind a new argument. we can remove 💥 from the changelog.

Reply:
+1. Also, instead of calling on all users to enhance their workflow with this new flag, could you explain why a user would want to use it? Reading https://github.com/overhangio/tutor/issues/967, my understanding is this most users will not need to use `-c`; rather, only plugin developers or other advanced users will need it, as they are the ones who will need to clean files out of their environment.

Reply:
> It's not a breaking change.

Oops. My bad.

> could you explain why a user would want to use it?

How about this?
- [Feature] Added `-c` or `--clean` option to `tutor config save`: For plugin developers and advanced users, this option cleans the `env/` folder before saving, ensuring a fresh environment for testing and development.

Reply:
Perfect 🚀

Reply:
Done